### PR TITLE
#trivial Fix tooltip eslint issues

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?

As the title says

#### What problem is this solving?

Related issue #1082 

#### Types of changes

- [x] Bugfix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Tooltip/Portal.tsx [hunk 1]
```diff
@@ -63,12 +58,8 @@ const Portal = (props: PropTypes.InferProps<typeof propTypes>, ref) => {
     }
   }, [onRendered, mountNode])
 
-  return mountNode ? ReactDOM.createPortal(children, mountNode) : mountNode
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (thayannevls) [inline react/components/Tooltip/Portal.tsx:61]: Maybe you can resolve this by adding the condition `mountNode && mountNode instanceof HTMLElement`

File: react/components/Tooltip/TooltipPopup.tsx [hunk 2]
```diff
@@ -1,6 +1,10 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable max-params */
+/* eslint-disable @typescript-eslint/no-use-before-define */
```
Comment (thayannevls) [inline react/components/Tooltip/TooltipPopup.tsx:4]: Does it exist another issue to resolve this error?

Comment (matheusps) [reply react/components/Tooltip/TooltipPopup.tsx:4]: Not yet, I'll create it after merges (like the lint issues one).

File: react/components/Tooltip/Portal.tsx (hunk 1 above)
Comment (matheusps) [reply react/components/Tooltip/Portal.tsx]: Solved! thanks

File: react/components/Tooltip/index.tsx [hunk 3]
```diff
@@ -1,12 +1,36 @@
 import PropTypes from 'prop-types'
-import React, { FC, cloneElement, Children } from 'react'
+import React, { FC, cloneElement, Children, ReactElement } from 'react'
 
 import TooltipPopup, { Position, Size } from './TooltipPopup'
 import { useTooltip, Trigger } from './hooks'
 
-const propTypes = {
+interface Props {
+  children: ReactElement
```
Comment (estacioneto) [inline react/components/Tooltip/index.tsx:8]: How about...

```suggestion
  children: ReactNode
```

File: react/components/Tooltip/TooltipPopup.tsx [hunk 4]
```diff
@@ -119,7 +103,12 @@ const TooltipPopup: FC<PropTypes.InferProps<typeof propTypes>> = ({
   )
 }
 
-const getStyles = (childRect, popupRect, position, fallbackPosition) => {
+const getStyles = (
+  childRect: null,
+  popupRect: null,
```
Comment (estacioneto) [inline react/components/Tooltip/TooltipPopup.tsx:108]: 🤔

Comment (matheusps) [reply react/components/Tooltip/TooltipPopup.tsx]: changed to any

**Pull request merged**